### PR TITLE
Allow closing modal manually

### PR DESCRIPTION
**Pull request by horans**
By adding new prop "okToClose", new method "close()" and modifying method "ok()", will allow a manual close action after "ok" event has been emitted.
This will ideal for scenario such as extra validation is needed for inputs on the modal.

### CODE REVIEW

File: src/components/vuestic-components/vuestic-modal/VuesticModal.vue [hunk 1]
```diff
@@ -108,7 +112,7 @@
       },
       cancel () {
         this.$emit('cancel')
-        this.show = false
+        void(this.okToClose ? this.show = false : null)
```
Comment (jean-moldovan) [inline src/components/vuestic-components/vuestic-modal/VuesticModal.vue]: Isn't it suppose to go to `close` method?

```
      close () {
        if (this.okToClose) {
          this.show = false
        }
      }
```

P.S. Run `npm run lint` to lint your changes locally

**Comment by horans** [comment]
`okToClose` is designed for advanced validation after `ok()`. i.e., a modal with form/inputs.
so i don't think we need it for `cancel()`.

**Comment by horans** [comment]
my eslint is not showing any errors.
very limited codes had been changed, so i assumed it was something to do with extra comma in the end?

**Comment by jean-moldovan** [comment]
yes, did you try running `npm run lint`?
we use StandardJS code style

**Comment by horans** [comment]
i am using customized rules for the project, and nothing come after `npm run lint`:

> vuestic-admin@1.1.1 lint xxxx
> eslint --ext .js,.vue src

i have updated codes according to the style of the rest anyway.

**Comment by smartapant** [comment]
@horans please resolve the conflicts so we could recheck PR

**Comment by horans** [comment]
@smartapant sorry for late reply due to chinese new year holiday. i am new to the community, and the "resolve conflicts" button is disabled. does it mean i need to clone it to local and resolve it by client/command?

**Comment by smartapant** [comment]
@horans Hi!

Try merging freshest master and update the PR

**Comment by horans** [comment]
@smartapant please check if this pull request is ok now, thanks!

**Pull request closed without merging**